pyproject.toml

```diff
@@ -1,7 +1,95 @@
-[tool.pyright]
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools]
+package-dir = {"" = "src"}
+packages = [
+    "gfn",
+]
+
+[tool.setuptools.package-data]
+"*" = [
+    "*.txt"
+]
+
+[project]
+name = "gfn"
+version = "0.2.0"
+
+authors = [
+  { name="Salem Lahou", email="lahlosal@mila.quebec" },
+  { name="Joseph Viviano", email="joseph@viviano.ca" },
+  { name="Victor Schmidt", email="schmidtv@mila.quebec" },
+]
+
+description = "A Python package for GFlowNets"
+readme = "README.md"
+requires-python = ">=3.10"
+
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+]
+
+dependencies = [
+    "einops",
+    "gymnasium",
+    "numpy",
+    "torch",
+    "torchtyping",
+]
 
+
+[project.optional-dependencies]
+wandb = [
```

Comment:
IIUC, naming this `wandb` means that `pip install .[wandb]` installs the extra dependencies. Wouldn't it make more sense then to call it `scripts` or something like this ?

Reply:
yeah that makes sense - I called it scripts so people don't think they are only installing `wandb`